### PR TITLE
prep for netbricks and some generic needs, e.g. tcpdump

### PR DESCRIPTION
**Pull request by zeeshanlakhani**

### CODE REVIEW

File: Dockerfile.sandbox [hunk 1]
```diff
@@ -12,22 +12,30 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     curl \
     gcc \
     git \
+    gdb \
```
Comment (seancribbs) [inline Dockerfile.sandbox]: What do we need `gdb` et al for?

Comment (zeeshanlakhani) [reply Dockerfile.sandbox]: debugging on the container (need gdb + gdbserver). We'll probably need it one day :) @seancribbs (works well w/ rust and c).

File: dpdk/Dockerfile [hunk 2]
```diff
@@ -9,7 +9,9 @@ ENV RTE_ARCH x86_64
 ENV CFLAGS "-g3 -Wno-error=maybe-uninitialized -fPIC"
 
 RUN apt-get update && apt-get install -y \
+    apt-utils \
```
[the remark this answers is not in the document]
Comment (zeeshanlakhani) [reply dpdk/Dockerfile]: gets rid of some specific dpdk preconfig errors.

**Comment by seancribbs** [comment]
:shipit:

**Pull request merged**